gprofiler/utils/__init__.py

```diff
@@ -351,7 +353,7 @@ def pgrep_exe(match: str) -> List[Process]:
         return procs
 
 
-def pgrep_maps(match: str) -> List[Process]:
+def pgrep_maps(match: str, ignore_permission_errors: bool = False) -> List[Process]:
```

Comment:
Let's remove this parameter, and instead - call `is_root` here to decide (instead of having all callers forced to call this and pass as an argument)